Formula/qt6.rb

```diff
@@ -0,0 +1,172 @@
+class Qt6 < Formula
+  desc "Cross-platform application and UI framework"
+  homepage "https://www.qt.io/"
+  url "https://download.qt.io/official_releases/qt/6.0/6.0.1/single/qt-everywhere-src-6.0.1.tar.xz"
+  sha256 "d13cfac103cd80b216cd2f73d0211dd6b1a1de2516911c89ce9c5ed14d9631a8"
+  license all_of: ["GFDL-1.3-only", "GPL-2.0-only", "GPL-3.0-only", "LGPL-2.1-only", "LGPL-3.0-only"]
+
+  depends_on "cmake" => :build
```

Comment:
```suggestion
  depends_on "cmake" => [:build, :test]
```
Please test this formula locally first before pushing another change, thanks. Also, please call the formula `qt@6`.